### PR TITLE
include_top, pooling args for squeezenet, and a jupyter notebook example

### PR DESCRIPTION
**Pull request by jeremykawahara**
Extends SqueezeNet to use the include_top and pooling args like other pretrained architectures do.

Includes a jupyter notebook example showing SqueezeNet in use.

### CODE REVIEW

**Comment by mweber7** [comment]
Squeezenet transfer learning

I am having some trouble implementing transfer learning using Squeezenet for the dogs/cats example described on the Keras blog.

Here is what I have tried:

model = SqueezeNet(weights='imagenet', include_top=False)

for layer in model.layers:
    layer.trainable = False

#Add custom layers for training
x = model.output
x = Dropout(0.5, name='drop9')(x)
x = Convolution2D(classes, (1, 1), padding='valid', name='conv10')(x)
x = Activation('relu', name='relu_conv10')(x)
x = GlobalAveragePooling2D()(x)
x = Dense(2, activation="softmax")(x)

model_final = Model(inputs=model.input, outputs=x)

This does not appear to work as the training accuracy and loss stay the same for each epoch with the accuracy being 50%.  Any idea what the issue is?  Do I need to preprocess the image data?  If so, how do I do that?
